www.foia.gov/_sass/modules/_hero_unit.scss

```diff
@@ -0,0 +1,78 @@
+.usa-hero {
+  background-image: none;
+  padding: 0;
+  @media screen and (min-width: $small-screen) {
+    background-image: url('../assets/Usdepartmentofjustice-1.jpg');
```

Comment:
Not seeing this file in the PR, are you going to leave it out until you get final optimized image from design?

Reply:
FYI, `/assets/` is ignored by git because these are all compiled/built files. I think we should create a new `www.foia.gov/img/` to put new images like this in.